config/data/paragonUtils.js

```diff
@@ -0,0 +1,153 @@
+const path = require('path');
```

Comment:
[proposal]: I propose to provide for the scalability that may occur over time and divide this file into several. This will improve the code's readability and give other developers a hint to separate the code semantically.
Please let me know what your thoughts are.

Reply:
I understand your point, but I'm not entirely convinced to split this file, the structure is clear and has an order easy to follow. Also, it's focused on just one specific topic.